angular.js

```diff
@@ -53,12 +58,28 @@ module.exports = {
 
         '@angular-eslint/no-attribute-decorator': 'error',
 
+        '@angular-eslint/no-forward-ref': 'off',
```

Comment:
Why is it necessary to declare all the rules? Shouldn't semver address this? It's useful to keep the scope of the ruleset to the stuff we care about.

Reply:
We cared enough to not enable it. Capturing it here shows that we evaluated it and made a decision. Also makes maintenance easier. During updates one can print the properties and see what is not captured here. That's how I found the other two rules that were not enabled during update.

Reply:
It might actually be a good idea to include a comment describing why the rule was not enabled.

Reply:
I like the rationale of adding comments to document why rules weren't enabled. I added a few notable ones from #45 as suggestions, but I don't think we need to be comprehensive about it in this PR.

Reply:
Was enough of a push for me to audit the rest. All the existing rules configured to 'off' should have justification and all the rules outside the recommended sets (for typescript and angular) should be captured in their respective config.

We rely on airbnb for the javascript / base rule configuration as evaluating all the base JS rules would be another large task, so not all rules from the base eslint rule set are captured in our ruleset.